bin/build.sh

```diff
@@ -1,73 +1,81 @@
 #!/bin/bash
 
-# Makes it so that all commands are displayed as they get executed
-set -x
-
-# WCA url and absolute path to WCA logo file
 wca_url="https://www.worldcubeassociation.org/"
 wca_docs_url="https://documents.worldcubeassociation.org/"
+# Absolute paths to the logo file, the main stylesheet and the edudoc header
 logo=$(realpath "./assets/WCAlogo_notext.svg")
-# Absolute paths to the stylesheets and the edudoc header
-stylesheet=$(realpath "./assets/style.css")
-edudoc_stylesheet=$(realpath "./assets/edudoc-style.css")
+main_stylesheet=$(realpath "./assets/style.css")
 edudoc_header=$(realpath "./assets/edudoc-header.html")
 # Current date
 compile_date=$(date '+%Y-%m-%d')
 
-# Function for converting Markdown files from a given folder into PDF files using the given stylesheet
-# The first argument ($1) is the folder and the second argument is the stylesheet ($2)
+# Function for converting Markdown files from a given directory into PDF files.
+# The first argument ($1) refers to the directory name (e.g: documents or edudoc).
 convert_to_pdf() {
+  rm -rf "build/$1"
```

Comment:
There is a "delete" step at the very bottom of the build script, and because of that, individually removing PDFs here wasn't necessary before. Why is it necessary now?

Reply:
I found myself sometimes waiting for a specific file to build, and this made it easier to spot it when it wasn't buried in .md and .html files, so I think this is a little better.